friends/src/components/Login.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import axios from "axios";
+import { useState } from "react"
+
+
+const Login = () => {
+
+    const [ creds, setCreds] = useState({username: "", password: ""});
+
+    const handleChange = e => {
+        setCreds({...creds, [e.target.name]: e.target.value});
+    };
+
+    const login = e => {
+        e.preventDefault();
+        axios
+        .post("http://localhost:5000/api/login", creds)
```

Comment:
I noticed you're not using the axiosWithAuth and passing the baseUrl through the axiosWithAuth call. This was something I think should have been here. See if you can add this to make sure you're authenticating the login.